gen-models/src/operations.rs

```diff
@@ -309,15 +309,54 @@ impl FileAddition {
         HashId(calculate_hash(&combined))
     }
 
+    fn normalize_file_paths(conn: &Connection, file_path: &str) -> (String, String) {
```

Comment:
this was added because files were being saved like `../fixtures/simple.fa`. It normalizes file paths relative to the .gen dir

Reply:
:+1: